packages/lab/stories/slider/slider.qa.stories.tsx

```diff
@@ -0,0 +1,96 @@
+import { Slider } from "@salt-ds/lab";
+import type { Meta, StoryFn } from "@storybook/react";
+import { QAContainer, type QAContainerProps } from "docs/components";
+
+export default {
+  title: "Lab/Slider/Slider QA",
+  component: Slider,
+} as Meta<typeof Slider>;
+
+export const Default: StoryFn<QAContainerProps> = (props) => {
```

Comment:
Do we need separate stories? This makes changes to Slider 4x expensive in Chromatic